scripts/verify-mdlint.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/bash
```

Comment:
Can we do this without requiring bash? Plain POSIX may work better for developers using Windows.

Reply:
Sure. we don't have to be on bash shell I believe.